src/app/lib/config/services/sport.js

```diff
@@ -0,0 +1,281 @@
+import {
+  C_SPORT_YELLOW,
+  C_MIDNIGHT,
```

Comment:
When I run this locally I get the following error:
![image](https://user-images.githubusercontent.com/9645462/105013042-be28a280-5a36-11eb-9a82-f362c8a5c937.png)
Did you do a PR to add colours in psammead @jimjohnsonrollings ? That will need integrating before this can be merged.

Reply:
Am I right in saying `C_MIDNIGHT` should be used for the blue links in the navigation? 
![image](https://user-images.githubusercontent.com/9645462/105014096-e9f85800-5a37-11eb-9b92-20481a35a2ed.png)

Reply:
Good spot - this is because it should be `C_MIDNIGHT_BLACK`!

Reply:
<img width="849" alt="Screenshot 2021-01-19 at 09 43 13" src="https://user-images.githubusercontent.com/8124899/105016615-c1be2880-5a3a-11eb-85f7-adee8c797e35.png">
Sorted